code/implementation/docker/machine/pom.xml

```diff
@@ -39,5 +39,10 @@
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
The AuthServiceLauncher is importing some Auth related DynamicStringProperties (for getReloadSettings()) from io.cattle.platform.iaas.api.auth.SecurityConstants and  io.cattle.platform.iaas.api.auth.integration.external.ServiceAuthConstants;